### ISSUE TITLE
SSL version mismatch on Arch Linux

### ISSUE THREAD

**Issue by dlukes** [fman-community/fman#189]
Hi,

fman is failing to launch on Arch Linux, presumably because of an SSL version mismatch -- this is what I got by running `/opt/fman/fman`:

```
Traceback (most recent call last):
  File "fman/main.py", line 1, in <module>
  File "/home/michael/dev/fman/venv/lib/python3.5/site-packages/PyInstaller/loader/pyimod03_importers.py", line 389, in load_module
  File "fman/impl/application_context.py", line 18, in <module>
  File "/home/michael/dev/fman/venv/lib/python3.5/site-packages/PyInstaller/loader/pyimod03_importers.py", line 389, in load_module
  File "fman/impl/signal_.py", line 1, in <module>
  File "/home/michael/dev/fman/venv/lib/python3.5/site-packages/PyInstaller/loader/pyimod03_importers.py", line 573, in load_module
ImportError: /opt/fman/libssl.so.1.0.0: version `OPENSSL_1.0.2' not found (required by /usr/lib/libQt5Network.so.5)
Failed to execute script main
```

Please let me know if I can be of any help in further diagnosing the issue :)

**Comment by mherrmann** [comment]
Thank you David. I'm aware of the issue and will look into it in the coming days.

**Comment by mherrmann** [comment]
(fman's Arch package is currently built on Ubuntu. The reason for the error (I think) is that Ubuntu's SSL version isn't compatible with Arch's latest one. I believe the solution to this will be to build fman for Arch - on Arch. If you have a better idea how the problem could be solve, that would be very interesting. If not, don't worry.)

**Comment by dlukes** [comment]
Thanks for the response, Michael! I guess that will be the simplest solution... I suppose you could also build fman on Ubuntu with a custom installed `libssl`, but that would require to manually keep track of which SSL version is the correct one for Arch.

The trouble with Arch is that since it's a rolling release distro, the library versions might change quite unexpectedly from day to day, with some users possibly lagging behind in the updates. So the most foolproof solution would probably be either to rely on the system version of `libssl` (no idea if that's doable for fman) or conversely, to bundle appropriate versions of all libraries that depend on `libssl` (cf. `libQt5Network` in the error output above, but there may be more...?) with fman (again, no idea if that's doable in practice).

**Comment by mherrmann** [comment]
I'm just in the process of trying to use Docker to build fman on Arch "natively". We'll see how it goes :)

**Comment by mherrmann** [comment]
Fixed in version 0.5.4.
